pkgs/by-name/te/text-engine/package.nix

```diff
@@ -40,19 +34,19 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    libadwaita
     json-glib
```

Comment:
would it make sense to apply https://github.com/mjakeman/text-engine/pull/42/ as a patch and remove json-glib?